packages/components/src/local/route/helpers/history-routes-for.ts

```diff
@@ -0,0 +1,38 @@
+import { LatLng } from 'leaflet'
+
+import SergeHex from '../../mapping/types/serge-hex'
+import SergeGrid from '../../mapping/types/serge-grid'
+import hexNamed from '../../hex-grid/helpers/hex-named'
+import { RouteData } from './planned-routes-for'
```

Comment:
```suggestion
import { RouteData,  LENGTH_OF_TRIMMED_LINE } from './planned-routes-for'
```
Out of curiosity, why does LENGTH_OF_TRIMMED_LINE not conform to our variable name pattern? The capitalisation and snake_case thing used to be because JS couldn't identify a difference between variables and constants but it can now so I see no benefit to this.

Reply:
oh, it's an old Java practice for highlighting a constant.  I'll change it.